src/components/shared/Header.tsx

```diff
@@ -30,156 +33,191 @@ export const Header = () => {
-  return (
-  <div className="App-title col-auto py-3 px-0">
-    <Link to={withLocaleUrl("Explore")} className="flex align-items-center">
-      <img src={Img} width={23} height={23} alt="" />
-      <div className="col-auto px-2 align-middle" >SeroTracker</div>
-    </Link>
-  </div>)
-}
-
-const mobileNav = (partnershipDrownDownActive: boolean, handleClick: any, changeLanguages:any, language: any) => {
-  return (
-  <Dropdown icon='bars' direction='left' simple>
-  <Dropdown.Menu>
-    <Dropdown.Item>
+  const AppTitle = () => {
+    return (
+    <div className="App-title col-auto py-3 px-0">
+      <Link to={withLocaleUrl("Explore")} className="flex align-items-center">
+        <img src={Img} width={23} height={23} alt="" />
+        <div className="col-auto px-2 align-middle" >SeroTracker</div>
+      </Link>
+    </div>)
+  }
+  
+  const mobileNav = (partnershipDrownDownActive: boolean, languageDropdownActive: boolean, handlePartnershipClick: any, handleLanguageClick: any, changeLanguages:any, language: any) => {
+    return (
+    <Dropdown icon='bars' direction='left' simple>
+    <Dropdown.Menu>
+      <Dropdown.Item>
+        <Link to={withLocaleUrl("Explore")}>
+            {Translate('Explore')}
+        </Link>
+      </Dropdown.Item>
+      <Dropdown.Item>
+        <Link to={withLocaleUrl("Analyze")}>
+          {Translate('Analyze')}
+        </Link>
+      </Dropdown.Item>
+      <Dropdown.Item>
+        <Link to={withLocaleUrl("Data")}>
+          {Translate('Data')}
+        </Link>
+      </Dropdown.Item>
+      <Dropdown.Item>
+        <Link to={withLocaleUrl("Publications")}>
+          {Translate('Publications')}
+        </Link>
+      </Dropdown.Item>
+      <Dropdown.Item>
+        <Link to={withLocaleUrl("About")}>
+          {Translate('About')}
+        </Link>
+      </Dropdown.Item>
+      <Dropdown.Item>
+        <Accordion fluid>
+            <Accordion.Title
+              active={partnershipDrownDownActive}
+              index={0}
+              onClick={handlePartnershipClick}
+              className={'flex center-item py-0'}
+            >
+              {Translate("Partnerships")}
+              <Icon name='dropdown' className={`ml-5 ${partnershipDrownDownActive ? 'dropdown__icon' : 'dropdown__icon--active'} `}/>
+            </Accordion.Title>
+            {renderPartnershipsAccordianContent(partnershipDrownDownActive)}
+          </Accordion>
+      </Dropdown.Item>
+      <Dropdown.Item>
+        <Accordion fluid>
+        <Accordion.Title
+              active={languageDropdownActive}
+              index={0}
+              onClick={handleLanguageClick}
+              className={'flex py-0'}
+            >
+              {state.language.toUpperCase()}
+              <Icon name='dropdown' className={`ml-5 ${languageDropdownActive ? 'dropdown__icon' : 'dropdown__icon--active'} `}/>
+              </Accordion.Title>
+              { state.language !== LanguageType.english ?    
```

Comment:
Confused as to why this is a !== LanguageType.english because wouldn't that mean 2/3 cases are always active (unless that is what is expected)?

Reply:
OHHH so if a certain language is active, it does not exist in the dropdown? Did I understand correct

Reply:
yeah exactly!